pkg/controllers/provisioningv2/rke2/bootstrap/controller.go

```diff
@@ -113,7 +113,15 @@ func (h *handler) getBootstrapSecret(namespace, name string, envVars []corev1.En
 			return nil, err
 		}
 
-		data, err := installer.LinuxInstallScript(context.WithValue(context.Background(), tls.InternalAPI, hasHostPort), base64.URLEncoding.EncodeToString(hash[:]), envVars, "")
+		is := installer.LinuxInstallScript
+		// This is only used by the vmwarevsphere node driver as of Jan 2022
+		// as we add multi-platform support to more node drivers
+		// this should continue to work based on changes in
+		// pkg/controllers/provisioningv2/rke2/provisioningcluster/template.go
```

Comment:
This comment is not necessary based on the code here.

Reply:
But this code doesn't care about which driver is being used. This code will "work" regardless of which driver is being used. There are other places where the code will break if windows is being used with a driver that doesn't support it. That is a more appropriate place for this comment.

Reply:
I changed the comment to a single line to indicate which node drivers support multiple OSs and sending a windows install script to. 

> 		// OS-specific logic is only implemented for the vmwarevsphere node driver as of Feb 2022